app/controllers/users_controller.rb

```diff
@@ -18,15 +18,17 @@ def create
   def show
     @user = User.find(params[:id])
     @activities = @user.activities.order('created_at DESC').paginate(page: params[:page], per_page: 5)
+    @learnwordscount =  Lesson.joins("INNER JOIN words ON lessons.category_id = words.category_id").where(lessons: {user_id: @user.id}).count
```

Comment:
@hironaooo 
You can use this method `current_user.words.count` by setting up an association in the `User` model:
`has_many :words, through: :lessons`